pkg/api/pluginproxy/ds_proxy.go

```diff
@@ -96,10 +96,12 @@ func (proxy *DataSourceProxy) HandleRequest() {
 		return
 	}
 
+	proxyErrorLogger := logger.New("userId", proxy.ctx.UserId, "orgId", proxy.ctx.OrgId, "uname", proxy.ctx.Login, "path", proxy.ctx.Req.URL.Path, "remote_addr", proxy.ctx.RemoteAddr(), "referer", proxy.ctx.Req.Referer())
```

Comment:
Couple of questions:

* Might `username` be less ambiguous than `uname`?
* Is there a reason for using snake case for `remote_addr` instead of camel case as e.g. `orgId`?

Reply:
Reusing same names from [here](https://github.com/grafana/grafana/blob/master/pkg/middleware/middleware.go#L73) and [here](https://github.com/grafana/grafana/blob/master/pkg/middleware/logger.go#L53).

Reply:
Trying to be keep consistence.